### ISSUE TITLE
Undefined FlashMessage constants in TYPO3 v13 LTS

### ISSUE THREAD

**Issue by schams-net** [t3rrific/aws-cloudfront-manager#9]
TYPO3 v13 LTS produces the following PHP exception:
```
Undefined constant TYPO3\CMS\Core\Messaging\AbstractMessage::WARNING
```

**Comment by schams-net** [comment]
Migration option:
```php
use TYPO3\CMS\Core\Type\ContextualFeedbackSeverity;
// provides: ContextualFeedbackSeverity::WARNING
```
See [TYPO3 documentation](https://docs.typo3.org/m/typo3/reference-coreapi/13.4/en-us/ApiOverview/FlashMessages/FlashMessagesApi.html)

**Comment by schams-net** [comment]
Resolved with commit ID e3d8e3c2e34fd55dc400aa3f8bd254a3ca550a2d.
